yield_plugin/src/yield_plugin.cpp

```diff
@@ -385,14 +392,19 @@ namespace yield_plugin
         double dt = (2 * original_traj_downtracks[i]) / (current_speed + prev_speed);
         jmt_tpp = original_tp.trajectory_points[i];
         jmt_tpp.target_time =  rclcpp::Time(jmt_trajectory_points[i-1].target_time) + rclcpp::Duration(dt*1e9);
+        std::cout << "non-empty x: " << jmt_tpp.x << ", y:" << jmt_tpp.y << ", t:" << std::to_string(rclcpp::Time(jmt_tpp.target_time).seconds()) << std::endl;
```

Comment:
We should be using ROS's logging features instead of directly printing to `std::cout`.